packages/Webkul/Core/src/Rules/CommaSeparatedInteger.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Webkul\Core\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class CommaSeparatedInteger implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        if (! $this->isCommaSeparatedInteger($attribute, $value)) {
+            $fail('core::validation.comma-separated-integer')->translate();
+        }
+    }
+
+    /**
+     * Determine if the value is comma separated integer.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function isCommaSeparatedInteger($attribute, $value)
+    {
+        $integerValues = explode(',', $value);
+
+        foreach ($integerValues as $integerValue) {
+            if (! preg_match('/^[0-9]+$/', $integerValue)) {
```

Comment:
@devansh-webkul Preg_match is slower. we can use just is_int to validate if it's an integer. Your thoughts?

Reply:
No need, that won't cause that much impact. Earlier we are using but that function causing problems. So we are avoiding `is_int`. If you want you can customize on your project.